src/common/TALENTS/priest.ts

```diff
@@ -2,14 +2,6 @@
 import { Talent } from './types';
 
 const talents = {
-  ABOMINATION_LIMB_TALENT: {
-    id: 383269,
-    name: 'Abomination Limb',
-    icon: 'ability_maldraxxus_deathknight',
-    maxRanks: 1,
-    entryIds: [96177],
-    definitionIds: [{ id: 101179, specId: 258 }],
-  },
```

Comment:
if anyone is curious why the priest file is -4,102 lines, its stuff like this from the previous bad raidbots data

Reply:
HUH

Reply:
there was a bunch of duplicated data showing the shadow priest spec id. @ljosberinn knows all about it

Reply:
shadow is so meta, even rogue and dk got a new shadow spec (and others)